### ISSUE TITLE
a11y: decorative image contains alt text

### ISSUE THREAD

**Issue by Danielle254** [kepsteen/great-harvest-landing-page#40]
Decorative images should have empty alt text so that they are not processed by screen readers.

![Image](https://github.com/user-attachments/assets/ae44d3d4-f30a-4905-a2b7-23e7e1f6c94c)
